include/panda_torque_mpc/obstacle_params_parser.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include <Eigen/Dense>
+#include <boost/shared_ptr.hpp>
+#include <fstream>
+#include <memory>
+#include <vector>
+
+#include <pinocchio/fwd.hpp>
+#include <pinocchio/multibody/model.hpp>
+#include "pinocchio/algorithm/joint-configuration.hpp"
+#include "pinocchio/algorithm/geometry.hpp"
```

Comment:
```suggestion
#include <pinocchio/algorithm/joint-configuration.hpp>
#include <pinocchio/algorithm/geometry.hpp>
```
Includes with `""` are for local files, while with `<>` are for system-wide libraries. With CMake it gets more complicates as often both cases are correctly recognized, but the practice is to use `<>`